WalletWasabi.Gui/Dialogs/CannotCloseDialogViewModel.cs

```diff
@@ -96,7 +96,8 @@ public override void OnOpen()
 		{
 			Disposables = Disposables is null ? new CompositeDisposable() : throw new NotSupportedException($"Cannot open {GetType().Name} before closing it.");
 
-			CancelTokenSource = new CancellationTokenSource().DisposeWith(Disposables);
+			using var cancellationTokenSource = new CancellationTokenSource();
+			CancelTokenSource = cancellationTokenSource.DisposeWith(Disposables);
```

Comment:
This is false again. `cancellationTokenSource` will be disposed with `Disposables`.